### ISSUE TITLE
Pin and Suffering - rust code block should be shell session

### ISSUE THREAD

**Issue by CosmicHorrorDev** [fasterthanlime/feedback#182]
**Where's the problem at?**
Please link to the article

https://fasterthanli.me/articles/pin-and-suffering

**No, seriously, where is it?**
Copy/paste the paragraph the problem is in, maybe highlight what's wrong?

...

This feels really dumb, but here goes:

```rust
// Rust code  <--
$ cargo run --quiet
thread 'main' panicked at 'not yet implemented', src/main.rs:19:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Yeah, well...

...

**Describe the issue and a fix if you know it**
In plain English, explain why this is wrong, and how to fix it (if you know how).

The rust code block should be a shell session block instead

**Comment by fasterthanlime** [comment]
> This feels really dumb

It isn't! Thanks for reporting this.

**Comment by CosmicHorrorDev** [comment]
That was actually part of the quote from the post 😅

Thanks for all the great content!
